automation/src/main/resources/puppet/manifests/dawg.pp

```diff
@@ -0,0 +1,97 @@
+class dawg::dawg {
+    class { 'java':
+        package => hiera('java.package', 'java-1.7.0-openjdk'),
+    }
+
+    $tomcatSrc = hiera('tomcat.src', 'http://archive.apache.org/dist/tomcat/tomcat-7/v7.0.73/bin/apache-tomcat-7.0.73.tar.gz')
+    tomcat::install { '/opt/tomcat':
+        source_url => $tomcatSrc,
+    }
+
+    file { '/etc/init.d/tomcat':
+        content => hiera('tomcat.init'),
+        mode => '0755',
+        require => Tomcat::Install['/opt/tomcat'],
+    }
+
+    exec {"genunit-tomcat":
+        path => '/sbin:/bin:/usr/sbin:/usr/bin',
+        onlyif => 'test -f /bin/systemctl',
```

Comment:
i don't think you have to have `systemctl`. You can always go back to `service tomcat start`